graspy/match/gmp.py

```diff
@@ -226,7 +220,7 @@ def fit(self, A, B, seeds_A=[], seeds_B=[]):
         B21T = np.transpose(B21)
         B22T = np.transpose(B22)
 
-        for i in range(self.n_init):
+        def forloop(init_num):
```

Comment:
can you think of a different name for this, maybe _fit_single_init() or something like that

Reply:
Sure.